lldb/source/Target/Statistics.cpp

```diff
@@ -153,6 +153,11 @@ TargetStats::ToJSON(Target &target,
                                       m_load_core_time.get().count());
     }
 
+    if (m_first_bt_time_set) {
```

Comment:
If we use a `std::once_flag` for this, we shlud just check if `m_active_time_to_first_bt` has a non-zero value